h2o-algos/src/main/java/hex/glrm/GLRMModel.java

```diff
@@ -392,6 +392,7 @@ public ModelMetricsGLRM scoreMetricsOnly(Frame frame) {
 
     GLRMScore gs = new GLRMScore(ncols, _parms._k, false).doAll(fullFrm);
     ModelMetrics mm = gs._mb.makeModelMetrics(GLRMModel.this, adaptedFr, null, null);   // save error metrics based on imputed data
+    mm._frame_checksum = frame.checksum();      // get the checksum of the original frame and not the one changed
```

Comment:
Michal: 

You are right.  That works and it is better than to expose the _frame_checksum.  Please have a look again.  Jenkins hates me and it is giving me problems.  I will make sure the tests are passed before I push.  Please do not worry about the test status.

Thanks,
Wendy